### PR TITLE
Require explicit output ports

### PR DESCRIPTION
**Pull request by wkozyra95**
- remove `port_range` option
- add `api_port` option that can take ranger or specific port
- add `port` option to the input pad that can take range or a specific port
- add `port` field to register_request
- fix `env` object to use strings as keys instead of atoms
- rename `lc_server_config` to `server_setup`
- remove the option to enable Chrome support
  - I don't want to force everyone to use binary with Chrome, so I prefer to disable that option for now.

### CODE REVIEW

File: lib/membrane/video_compositor/server_runner.ex [hunk 1]
```diff
@@ -1,10 +1,41 @@
...
+  require Membrane.Logger
+
+  alias Membrane.LiveCompositor
   alias Membrane.LiveCompositor.Request
 
-  @spec start_lc_server(:inet.port_number(), map()) :: :ok | :error
-  def start_lc_server(lc_port, env) do
+  @spec start_server(:inet.port_number() | LiveCompositor.port_range(), map()) ::
+          {:ok, :inet.port_number()} | {:error, err :: String.t()}
+  def start_server(port_or_port_range, env) do
```
Comment (wkozyra95) [inline lib/membrane/video_compositor/server_runner.ex:11]: video_compositor.ex is very large now, and this code was easy to extract

File: lib/membrane/video_compositor/output_options.ex [hunk 2]
```diff
@@ -30,6 +30,7 @@ defmodule Membrane.LiveCompositor.OutputOptions do
           width: Membrane.RawVideo.width_t(),
           height: Membrane.RawVideo.height_t(),
           id: Membrane.LiveCompositor.output_id(),
+          port: :inet.port_number(),
```
Comment (wkozyra95) [inline lib/membrane/video_compositor/output_options.ex:33]: if we simplify the pipeline setup this option will go to the pad

File: lib/membrane/video_compositor/request.ex [hunk 3]
```diff
@@ -22,16 +22,20 @@ defmodule Membrane.LiveCompositor.Request do
...
       type: :register,
-      entity_type: :input_stream,
+      entity_type: :rtp_input_stream,
       input_id: "#{input_id}",
-      port: input_port_number,
+      port:
+        case port do
+          {start, endd} -> "#{start}:#{endd}"
+          exact_port -> exact_port
+        end,
```
Comment (WojciechBarczynski) [inline lib/membrane/video_compositor/request.ex:38]: I think this looks better than putting the case statement inside the map declaration
```elixir
port = case port do ...
```

```elixir
%{
  port: port
}
```

**Review by WojciechBarczynski — APPROVED** [review]
LGTM

**Pull request merged**